### PR TITLE
Feature/99 cleanup

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#99
| License       | MIT

This PR proposes some corrections and improvements encountered while working on other issues.

### CODE REVIEW

File: src/CoreBundle/private/modules/CMSUpdateManager/CMSUpdateManager.class.php [hunk 1]
```diff
@@ -59,11 +59,18 @@ public function GetHtmlHeadIncludes()
     {
         $aIncludes = parent::GetHtmlHeadIncludes();
         $aIncludes[] = '<link href="'.TGlobal::GetPathTheme().'/css/modules/updateManager.css" rel="stylesheet" type="text/css" />';
-        $aIncludes[] = '<script src="'.TGlobal::GetStaticURLToWebLib('/javascript/modules/updateManager/updateManager.js').'" type="text/javascript"></script>';
 
         return $aIncludes;
     }
 
+    public function GetHtmlFooterIncludes()
```
Comment (UlrichKu) [inline src/CoreBundle/private/modules/CMSUpdateManager/CMSUpdateManager.class.php]: Looks reasonable.

But is there maybe a special reason (here) why this change is needed?

Comment (umulmrum) [reply src/CoreBundle/private/modules/CMSUpdateManager/CMSUpdateManager.class.php]: The CHAMELEON variable is now defined only in the footer and is required by the update manager, so the change is needed, yes.
